package.json

```diff
@@ -34,7 +34,7 @@
   "devDependencies": {
     "@emotion/react": "^11.11.4",
     "@emotion/styled": "^11.11.5",
-    "@mui/material": "^5.16.4",
+    "@mui/material": "^6.0.0",
```

Comment:
Why just 6.0.0? Why not 6.1.9 which is latest?

Reply:
Yes we can update to 6.1.9, i was following what documentation suggests [here](https://mui.com/material-ui/migration/upgrade-to-v6/#quality-of-life-improvements)